src/transformers/models/imagebind/processing_imagebind.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2024 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Image/Text processor class for ImageBind
+"""
+
+from typing import List, Optional, Union
+
+import numpy as np
+
+from ...image_utils import (
+    ImageInput,
+    VideoInput,
+)
+from ...processing_utils import AudioKwargs, ImagesKwargs, ProcessingKwargs, ProcessorMixin, Unpack
+from ...tokenization_utils_base import (
+    BatchEncoding,
+    PreTokenizedInput,
+    TextInput,
+)
+
+
+class ImageBindProcessorImagesKwargs(ImagesKwargs, total=False):
+    do_convert_rgb: bool
+    do_chunk: bool
+    chunk_duration: float
+    num_chunks: int
+    num_frames_per_chunk: int
+    fps: int
+
+
+class ImageBindProcessorAudioKwargs(AudioKwargs, total=False):
+    do_normalize: Optional[bool]
+    mean: Optional[float]
+    std: Optional[float]
+    do_chunk: Optional[bool]
+    chunk_duration: Optional[float]
+    num_chunks: Optional[int]
```

Comment:
kinda wondering why we don't use them in the FeatureExtractor as well!

Reply:
@molbap

Reply:
We could - the main appeal for standardizing the processor was for pipeline + api use, but there's no harm in doing it at all call levels!

Reply:
@molbap @ArthurZucker If this change is to be done then should we accompany it in this PR?

Reply:
Yep we can!